### PR TITLE
python3Packages.pygame-gui: init at 0.5.7

### PR DESCRIPTION
**Pull request by emilytrau**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 130328` at 62335614 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.pygame-gui</li>
    <li>python39Packages.pygame-gui</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Consider adding a comment explaining the purpose of this patch on the line preceeding.
Near pkgs/development/python-modules/pygame-gui/default.nix:24:5:
```
   |
24 |     (substituteAll {
   |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Consider adding a comment explaining the purpose of this patch on the line preceeding.
Near pkgs/development/python-modules/pygame-gui/default.nix:28:5:
```
   |
28 |     ./skip-clipboard-tests.patch
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -5992,6 +5992,8 @@ in {
 
   pygame_sdl2 = callPackage ../development/python-modules/pygame_sdl2 { };
 
+  pygame-gui = callPackage ../development/python-modules/pygame-gui { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:5995]: Is this a library or an end user program?

File: pkgs/development/python-modules/pygame-gui/default.nix [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+}:
+
+buildPythonPackage rec {
+  pname = "pygame-gui";
+  version = "0.5.7";
+
+  src = fetchFromGitHub {
+    owner = "MyreMylar";
+    repo = "pygame_gui";
+    rev = "v_057";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pygame-gui/default.nix:16]:
```suggestion
    rev = "v_${lib.replaceStrings ["."] [""] version}";
```

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (angustrau) [reply pkgs/top-level/python-packages.nix:5995]: This is a library

File: pkgs/development/python-modules/pygame-gui/default.nix (hunk 2 above)
Comment (angustrau) [reply pkgs/development/python-modules/pygame-gui/default.nix]: Done

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -5990,6 +5990,8 @@ in {
 
   pygame = callPackage ../development/python-modules/pygame { };
 
+  pygame_gui = callPackage ../development/python-modules/pygame_gui { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:5993]:
```suggestion
  pygame-gui = callPackage ../development/python-modules/pygame-gui { };
```
new packages should be normalized and old ones eventually converted.

File: pkgs/development/python-modules/pygame_gui/default.nix [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pygame_gui";
+  version = "0.5.7";
+
+  src = fetchFromGitHub {
+    owner = "MyreMylar";
+    repo = pname;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pygame_gui/default.nix:15]:
```suggestion
  pname = "pygame-gui";
  version = "0.5.7";

  src = fetchFromGitHub {
    owner = "MyreMylar";
    repo = "pygame_gui";
```

File: pkgs/top-level/python-packages.nix (hunk 3 above)
Comment (angustrau) [reply pkgs/top-level/python-packages.nix]: Apologies. I have reverted it

**Pull request merged**